ynab_to_ledger.rb

```diff
@@ -11,13 +11,27 @@ def main
 end
 
 def ledger_entry(row)
+  [row["Inflow"], row["Outflow"]].each do |amount|
+    begin
+      Float(amount)
+    rescue
+      STDERR.puts "Invalid inflow or outflow: #{amount}, expected to be in format 0.00"
+    end
+  end
+
   inflow = blank_if_zero(row["Inflow"])
   outflow = blank_if_zero(row["Outflow"])
 
   return if inflow == "" && outflow == ""
 
   month, day, year = row["Date"].split("/")
-
+  
+  begin
+    Date.new(Integer(year, 10), Integer(month, 10), Integer(day, 10))
```

Comment:
If you're parsing as a Date, do you still need to first parse as Integer?

Reply:
Yes, to avoid this error: `invalid day (not numeric)`